src/Apps/Components/AppToasts.tsx

```diff
@@ -29,3 +32,16 @@ export const AppToasts: React.FC = () => {
     </Box>
   )
 }
+
+/**
+ * Check if Toasts' bottom offset needs to be adjusted due to presence of
+ * cookie consent banner.
+ */
+function useBottomOffset(): number {
+  const { isDisplayable } = useConsentRequired()
+
+  if (isServer || !isDisplayable) return 0
```

Comment:
The use of `isServer` here may cause a server/client mismatch. We should set the height in a `useEffect`